CHANGES.rst

```diff
@@ -1,13 +1,13 @@
 Change Log
 ==========
 
-Unreleased
-----------
+6.3.1 (2019/06/10)
+------------------
 
-**Changed**
+**Removed**
 
-* Removed ``FullnameMixin.kind``
-* Removed ``SubredditListingMixin.gilded()``
+* ``FullnameMixin.kind``
+* ``SubredditListingMixin.gilded()``
```

Comment:
Can we add that this was _supposed_ to be removed in 6.0 after being deprecated in 5.X.X, but we forgot?

Reply:
Yes.